.github/workflows/realm-dart-linux.yml

```diff
@@ -2,19 +2,22 @@ name: Realm Dart for Linux
 
 on:
   push:
-    branches: 
+    branches:
       - master
-    # Do not run if change is in these paths
     paths-ignore:
       - 'flutter/**'
       - 'generator/**'
+      - '.github/**'
+      - '!.github/workflows/realm-dart-linux.yml' # except this one
```

Comment:
as above. seems counter intuitive not to run the linux CI when it's script changes.

Reply:
The ! means this path is not ignored.. I'm ignoring changes to the other actions